src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/RazorProjectService.cs

```diff
@@ -210,7 +220,7 @@ private void ActOnDocumentInMultipleProjects(string filePath, Action<IProjectSna
         var textDocumentPath = FilePathNormalizer.Normalize(filePath);
         if (!_snapshotResolver.TryResolveAllProjects(textDocumentPath, out var projectSnapshots))
         {
-            projectSnapshots = new[] { _snapshotResolver.GetMiscellaneousProject() };
+            projectSnapshots = [_snapshotResolver.GetMiscellaneousProject()];
```

Comment:
Is it really worth keeping the sync version of this `ActOn` method? Seems like its only used in a couple of spots that could just return Task.CompletedTask from their lambdas, and it would be one less use of the sync GetMiscProject method. To say nothing of the logic duplication

Reply:
That's a fair point. I'll take a look and see if I can consolidate these.

Reply:
Done!